necessist/tests/necessist_db_absent/cli_dir_input.toml

```
@@ -0,0 +1,11 @@
+args = ["--no-sqlite", "--dump-candidates", "--framework=rust", "--root=fixtures/cli_dir_input", "fixtures/cli_dir_input/src/target_dir"]
```

```suggestion
args = ["--no-sqlite", "--dump-candidates", "--framework=rust", "--root=fixtures/cli_dir_input"]
```

Please see my "two tests" comment.